### ISSUE TITLE
Autocomplete

### ISSUE THREAD

**Issue by santheo** [dcwatson/DjangoTemplates.novaextension#1]
Thanks for making this!

I'm not finding the autocompletion to work. I type "if" and don't get any suggestions. Is there something I need to turn on?

-Sandy

**Comment by dcwatson** [comment]
No, I wrote the README optimistically - I haven't actually added any completions yet. I'd been sitting on this for a while waiting for Nova 4 to come out, and maybe jumped the gun a bit. I'll get completions in there soon.

**Comment by dcwatson** [comment]
FYI, I just pushed out version 1.1, which includes autocompletions for built-in templatetags and filters.

I'm not a big autocomplete user myself, so please let me know what you think. Specifically, I made them trigger *after* you've typed the tag opener (`{% `) or filter operator (`|`). I've seen some completions that simply trigger when you start typing a known tag/filter and insert the opener/closer around them, but this seems like it might interfere in a potentially mixed-language environment like a template? Also maybe just personal preference?

**Comment by djch** [comment]
Firstly, thanks for publishing an extension for Django templates.

I'm noticing that the built-in autocompletions for HTML (not to mention from Emmett) don't seem to be coming up on the Django Template syntax mode. Is there a way for you to include those alongside your own?
